# Motor speeds message from the motor sensors in Albatros motor board.
# Speeds are given according enumeration in motorboardctrl library.
# The units are percentage of nominal speed for input and rpm for output.

int32[4] speeds

